allennlp/training/optimizers.py

```diff
@@ -158,7 +157,6 @@ def from_params(cls, model_parameters: List, params: Params):  # type: ignore
     "rmsprop": (torch.optim.RMSprop, None),
     "adamax": (torch.optim.Adamax, None),
     "averaged_sgd": (torch.optim.ASGD, None),
-    "bert_adam": (BertAdam, None),
```

Comment:
@mhagiwara, FYI, looks like we should use `huggingface_adamW` instead of `bert_adam`.

Reply:
Do you know when this was added? I would like to change it to just `adamw`

Reply:
I don't know, but if `adamw` is a relatively standard / reasonable name, I'm totally fine with renaming it.  Also not hard to register it twice.

Reply:
Oh, at that rate, should also be fine to just have `bert_adam` use the new one, also, so we don't break anyone.

Reply:
It takes different arguments though

Reply:
@matt-gardner thanks for the info!